src/com/jalasoft/search_files/search/asset.py

```diff
@@ -0,0 +1,40 @@
+#import File
+import os
+
+class Asset (object):
```

Comment:
Please add a new line before of the Class definition line, it should contains 2 lines before of it